python/py-babel/Portfile

```diff
@@ -5,14 +5,14 @@ PortGroup           python 1.0
 
 name                py-babel
 python.rootname     Babel
-version             2.9.1
+version             2.10.3
 revision            0
 categories-append   devel
 platforms           darwin
 license             BSD
 supported_archs     noarch
 
-python.versions     27 35 36 37 38 39 310
+python.versions     37 38 39 310
```

Comment:
Since the latest version doesn't build for Python 2.7, 3.5, and 3.6, the port should fallback to the previous release for those cases.